go-controller/pkg/allocator/ipgenerator/ip_generator.go

```diff
@@ -1,4 +1,4 @@
-package clustermanager
+package ipgenerator
```

Comment:
What about a new main package uder `pkg` ? `pkg/generator` for this kind of things since they are not really allocating them, we can use that package udn things later on like join or masquerade `pkg/generator/udn/`

```golang
import(
   ipgenerator "github.com/ovn-org/ovn-kubernetes/pkg/generator/ip"
)

ipGenerator := ipgenerator.New()
```

Reply:
done. @qinqon PTAL